packages/data-extensions/README.md

```diff
@@ -0,0 +1,27 @@
+# `@shopify/data-extensions`
+
+This package contains the public type definitions and utilities needed to create a Shopify data extension.
+
+## What is a data extension?
+
+A data extension is a JavaScript-based module that can hook in to client-side behaviors on Shopify’s first party UI surface areas. It differs from a UI extension in that it does not establish a long-lived channel or render any UI on the surface. It runs once and returns data.
```

Comment:
Does it only run once? Don't we re-run if data changes?

Reply:
I'm not suggesting we watch changes. I just think the wording could be confusing. The data extension gets executed once for the provided input data. However, it also re-runs when the data changes like (ex. navigate to another product). We also re-run the function whenever the merchant navigates back to the same resource since we don't cache results currently. Maybe we say:

```suggestion
A data extension is a JavaScript-based module that can hook in to client-side behaviors on Shopify’s first party UI surface areas. It differs from a UI extension in that it does not establish a long-lived channel or render any UI on the surface. It is a stateless function that executes and returns data.
```